packages/ListBox/stories/examples/Showcase.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { boolean, text } from "@storybook/addon-knobs";
+import { Rule } from "storybook/assets/styles/common.styles";
+import * as characters from "../fixtures/characters";
+import ListBox from "../../src";
+
+const getKnobs = () => ({
+  hasImplicitAll: boolean("hasImplicitAll", false),
+  isDisabled: boolean("isDisabled", false),
+  placeholder: text("placeholder", "Select..."),
+  showFilter: boolean("Show Filter", false),
+  showFooter: boolean("Show Footer", false),
```

Comment:
Should we add `height`, `isInline`, and `size`?

Reply:
There were issues with `isInline, and size`. I talked to Nahum about `size` and it's because its not a controlled prop. It only works as like an "initial value". If i changed the prop it won't re-render with the new size